astro/src/content/docs/lifecycle/manage-users/verification/gate-accounts-until-user-phone-verified.mdx

```diff
@@ -0,0 +1,181 @@
+
+<img src="/img/docs/lifecycle/manage-users/verification/phone-template-customization.png" alt="Customizing the phone verification message template" width="1200" />
+
+## Configure the Application
+
+Navigate to <Breadcrumb>Applications</Breadcrumb>. Edit your application by clicking on the blue <InlineUIElement>Edit</InlineUIElement> button.
+
+Enable self service registration by selecting the <Breadcrumb>Registration</Breadcrumb> tab, and then scrolling to the <InlineUIElement>Self service registration</InlineUIElement> section.
+
+<Aside type="note">
+You can create a user with an unverified phone number in other ways than self service registration. However, self service registration is the simplest way to create a user without a verified phone number.
+</Aside>
+
+Enable self service registration and configure the displayed and required fields as desired (in particular, choose `Phone` for the `Login type`).
+
+<img src="/img/docs/lifecycle/manage-users/verification/application-enable-self-registration-phone.png" alt="Enabling application self registration" width="1200"  role="top-cropped bottom-cropped" />
+
+Next, navigate to the <Breadcrumb>OAuth</Breadcrumb> tab. Enable the `Authorization Code` grant.
+
+In addition, ensure you have entered a valid URL in the <InlineField>Authorized redirect URLs</InlineField> field. (The value of this redirect URL doesn't really matter for this tutorial. But for production use, ensure the code at that endpoint can exchange an authorization code for a token.)
+
+Save the application configuration. You'll be returned to the list of applications.
+
+Click on the green <InlineUIElement>View</InlineUIElement> button of your application to see the details of your application.
+
+<img src="/img/docs/lifecycle/manage-users/verification/view-application-details.png" alt="View application details to see the relevant URLs" width="1200" />
+
+Look for the <InlineField>Registration URL</InlineField>. It'll be something like `https://local.fusionauth.io/oauth2/register?client_id=85a03867-dccf-4882-adde-1a79aeec50df&response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Foauth-redirect`. Copy that value as you'll use it below.
+
+At the same time, find the <InlineField>OAuth IdP login URL</InlineField>, and copy that as well.
+
+## Test It Out
+
+Now, test out the functionality. Register a new account. Make sure to use a different browser or incognito mode so that the admin account you logged in with while configuring FusionAuth doesn't interfere with new user registration.
+
+Visit the registration URL copied above. You'll be prompted to register. Fill out the form with a valid phone number. After registration is complete, you'll receive a message prompting you to verify the phone number.
+
+Here is the message in the iOS interface:
+
+<img src="/img/docs/lifecycle/manage-users/verification/verification-sms-message-ios.png" alt="Verification message in iOS" width="1200" role="bottom-cropped" />
```

Comment:
figured an ios messages app might be best here but screenshot person can be the judge later on

Reply:
Do you mean macOS? Not sure we even need to mention the OS.

Reply:
whatever is best to show SMS messages.